algorithms-and-data-structures/basic-javascript/index040.js

```diff
@@ -0,0 +1,6 @@
+var xp = 0;
+var health = 100;
+var gold = 50;
+var currentWeapon = 0;
+
+// We've been declaring variables with the 'var' keyword. This makes a variable available everywhere. But it's better to scope variable as narrowly as possible. Change every var in the code to let so that the variables have local scope instead of global scope.
```

Comment:
Add `'` quotes to the `var` and `let` keywords like you did in the first sentence. Also, consider using a multi-line comment for the instructions.
```suggestion
/*
We've been declaring variables with the 'var' keyword.
This makes a variable available everywhere.
But it's better to scope variable as narrowly as possible.
Change every 'var' in the code to 'let' so that the variables have local scope instead of global * scope.
*/
```